port/win32/omrfiletext.c

```diff
@@ -181,7 +180,7 @@ omrfile_write_text(struct OMRPortLibrary *portLibrary, intptr_t fd, const char *
 
 	result = portLibrary->file_write(portLibrary, fd, (void *)buf, nbytes);
 
-	if (newBuf != stackBuf && newBuf != NULL) {
+	if ((stackBuf != newBuf)&& (NULL != newBuf)) {
```

Comment:
```suggestion
	if ((stackBuf != newBuf) && (NULL != newBuf)) {
```

Reply:
Fixed in 901ae9e35bb330b5ce4fc1f5d0ea8f4b7ded11ae.